Lib/asyncio/proactor_events.py

```diff
@@ -161,7 +163,6 @@ def _call_connection_lost(self, exc):
             # cure it, but maybe using DisconnectEx() would be better.
             if hasattr(self._sock, 'shutdown') and self._sock.fileno() != -1:
                 self._sock.shutdown(socket.SHUT_RDWR)
-            self._sock.close()
```

Comment:
This doesn't need to be removed else it'll leak the socket.

Reply:
Yep that was accidental. Thanks